hedera-node/src/main/java/com/hedera/services/store/tokens/HederaTokenStore.java

```diff
@@ -844,6 +847,43 @@ public MerkleUniqueToken getUniqueToken(final NftID id) {
 		return uniqueTokenSupplier.get().get(fromNftID(id));
 	}
 
+	@Override
+	public List<TokenNftInfo> getUniqueTokenInfosFor(AccountID id, long start, long end) {
```

Comment:
We should be returning `Pair<MerkleUniqueTokenId, MerkleUniqueToken>` to be consistent with Store <> State View methods